GameBoyPrinterEmulator/GameBoyPrinterEmulator.ino

```diff
@@ -159,14 +158,42 @@ void serialClock_ISR(void)
   Main Setup and Loop
 *******************************************************************************/
 
-void setup(void)
-{
+void setup(void) {
+
```

Comment:
Did you run this though some style linter? My typical code type is to have the opening parenthesis in a new line to make indentation easier to see.

Reply:
Oh, this is the automatic indentation of the Arduino IDE 2.0, sorry. Can you change this after merging or do you prefer I push a revised version ?